README.md

```diff
@@ -1,198 +1,48 @@
-			* [P0] Create a new talk on the future of #MLonCode and deliver it at least four times (existing confs and meetups)
-			* [P1] Make that workshop and talk available online
-			* [P1] Create two demos showing the power of our stack solving a “real problem”
-			* [P2] Determine what percentage of time source{d} engineers should spend working on docs/tutorials/talks/etc
-		* [P0] Make source{d} the lead of MLonCode research ([#42](https://github.com/src-d/okrs/issues/42))
-			* [P0] Create and maintain a research log with TL;DR summaries of research read
-			* [P1] Publish 6 ML blog posts in Q2
-			* [P2] Speak about ML on >4 relevant conferences
-			* [P2] Reach 1,000 cumulative upvotes of new blog posts on Reddit
-		* [P0] Enable sharing and publishing MLonCode models ([#44](https://github.com/src-d/okrs/issues/44))
-			* [P0] Define and design the product for model sharing (relates to
-			* [P1] Conduct user research with all key users (e.g.: key academic groups, users of our stack).
-		* [P0] Release updated content for sourced.tech ([#13](https://github.com/src-d/okrs/issues/13))
-			* [P0] Create a content strategy plan ([#35](https://github.com/src-d/okrs/issues/35))
-				* [P0] Editorial Environment Setup
-				* [P0] Customer Journeys
-				* [P0] Content Model
-				* [P0] Copywriting
-				* [P0] Metrics & Evaluation Analytics Setup
-				* [P1] Information Architecture
-				* [P2] Establish a Content Management Plan
-			* [P1] Update brand touchpoints according to content plan
-			* [P1] Apply new Visual Style to sourced.tech ([#32](https://github.com/src-d/okrs/issues/32))
-				* [P0] Visual Artifacts Study
-				* [P1] Research
-				* [P1] Chromatic Palete Study
-				* [P1] Typography Study
-				* [P1] Initial Proposals
-				* [P1] Pure css update PR
-			* [P1] Publish the CAT case-study development post on Medium ([#38](https://github.com/src-d/okrs/issues/38))
-				* [P0] Write a blog post regarding the process, methodologies and collaboration flow between teams on CAT
-	* Define DevRel organization ([#48](https://github.com/src-d/okrs/issues/48))
-		* [P0] Describe the metrics of devrel success
-		* [P0] Set monitoring up for all processes valuable to devrel
-		* [P1] Create a dashboard displaying those metrics
-		* [P0] Implement kanban processes in DevRel, follow the source{d} Engineering Methodology
-		* [P0] Set up biweekly sync-ups with Eiso, Marcelo, Maximo, and Fernanda
+# OKRs 2018 Q3
+
+## Bring 'Code as Data' to market with enterprises
+
+### gitbase & engine
+
+- [ ] Stable & Performance Release
+- [ ] Integration of gitbase with engine
+- [ ] New engine interface based on gitbase
+
+### Babelfish
+
+- [ ] Client/Library improvements, packaging, and stability
+- [ ] Iterate the higher level UAST abstractions to fix and maintain
+
+### Gemini
+
+- [ ] Release a new version with the integration of function similarity
+
+## Being a better company to work at
+
+- [ ] Regression Test, Benchmarks and Deploy platform (CD?)
```

Comment:
For which projects? Are any of ML included?

Reply:
Its a platform, to be used by the projects.

Reply:
Yes, any project that requires it. Implementation of actual benchmarks and regression tests are up to each team.